microsoft-edge/webview2/releasenotes.md

```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
+## 1.0.774-prerelease  
```

Comment:
Prerelease candidate is likely 773.
```suggestion
## 1.0.773-prerelease  
```

Reply:
Other version instances have to be updated too.

Reply:
Changed to 773!